### PR TITLE
add eCPRI class

### PR DESCRIPTION
**Pull request by zorrohahaha**
Signed-off-by: Bing Zhao <bingz@mellanox.com>

This is just a checklist to guide you. You can remove it safely.

**Checklist:**

-   [ ] If you are new to Scapy: I have checked <https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md> (esp. section submitting-pull-requests)
-   [ ] I squashed commits belonging together
-   [ ] I added unit tests or explained why they are not relevant
-   [ ] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)

> brief description what this PR will do, e.g. fixes broken dissection of XXX

> if required - short explanation why you fixed something in a way that may look more complicated as it actually is

> if required - outline impacts on other parts of the library

fixes #xxx (add issue number here if appropriate, else remove this line)

### CODE REVIEW

File: scapy/contrib/ecpri.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#
+# Scapy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Scapy. If not, see <http://www.gnu.org/licenses/>.
+
+# scapy.contrib.description = Multiprotocol Label Switching (MPLS)
```
Comment (raslandarawsheh) [inline scapy/contrib/ecpri.py:15]: I think you might want to remove these two lines

File: scapy/contrib/ecpri.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+             3: "generic-data-transfer",
+             4: "remote-memory-access",
+             5: "oneway-delay-measurement",
+             6: "remote-reset",
+             7: "event-indication",
+             8: "iwf-startup",
+             9: "iwf-operation",
+             10: "iwf-mapping",
+             11: "iwf-delay-control"}
+
```
Comment (raslandarawsheh) [inline scapy/contrib/ecpri.py:39]: don't we have the ecpri_common header as well ?

File: scapy/contrib/ecpri.py (hunk 1 above)
Comment (zorrohahaha) [reply scapy/contrib/ecpri.py:15]: > I think you might want to remove these two lines

Yes, I copy that file and forget to remove these two lines.

File: scapy/contrib/ecpri.py (hunk 2 above)
Comment (zorrohahaha) [reply scapy/contrib/ecpri.py:39]: > don't we have the ecpri_common header as well ?

I define this header as that in the ICMP format.

A common part for the type of the following bytes.
A variable part with different length will be parsed based on the type field in the common part.

**Pull request merged**